### ISSUE TITLE
OSEror Command ./configure failed with error code 126

### ISSUE THREAD

**Issue by dradux** [ekalinin/nodeenv#168]
I get the following error when creating a new env:
* nodeenv version: 0.13.6
* installed via pip
* os: debian (sid)
```
drad@thez:/media/pny64/apps/node$ nodeenv -v env
 * Install node (6.2.1)./bin/sh: 1: ./configure: Permission denied
/bin/sh: 1: ./configure: Permission denied
Traceback (most recent call last):
  File "/usr/local/bin/nodeenv", line 9, in <module>
    load_entry_point('nodeenv==0.13.6', 'console_scripts', 'nodeenv')()
  File "/usr/local/lib/python2.7/dist-packages/nodeenv.py", line 891, in main
    create_environment(env_dir, opt)
  File "/usr/local/lib/python2.7/dist-packages/nodeenv.py", line 732, in create_environment
    install_node(env_dir, src_dir, opt)
  File "/usr/local/lib/python2.7/dist-packages/nodeenv.py", line 608, in install_node
    build_node_from_src(env_dir, src_dir, node_src_dir, opt)
  File "/usr/local/lib/python2.7/dist-packages/nodeenv.py", line 575, in build_node_from_src
    callit(conf_cmd, opt.verbose, True, node_src_dir, env)
  File "/usr/local/lib/python2.7/dist-packages/nodeenv.py", line 461, in callit
    % (cmd_desc, proc.returncode))
OSError: Command ./configure --prefix=/media/pny64/apps/node/env failed with error code 126
drad@thez:/media/pny64/apps/node$
```

If I add the `--prebuilt` flag it works without an issue.

Also note, if I create an env on a local partition (e.g. under /home/user/apps) the env builds successfully.

I see the "Permission Denied" error in the dump but I have no idea why. If I try to execute ./configure in the /src/node-v6.2.1 I get `bash: ./configure: Permission denied`. Permissions on the file are:
```
drad@thez:/media/pny64/apps/node/env3/src/node-v6.2.1$ ll configure
-rwxr-xr-x 1 drad drad 42K Jun  2 17:53 configure
```
The /media/pny64 permissions are:
```
drwxrwxr-x  6 drad drad    67 Jun  8 10:04 pny64
```

FYI my /home partition is an ext4 partition while the partition that fails is an xfs partition. The user I am creating the nodeenv with owns both partitions.

**Comment by dradux** [comment]
FYI, if I use the `--prebuild` flag, the nodeenv is created and activates fine but is not properly setup.

```
drad@thez:/media/pny64/apps/node$ . env-6.2.1/bin/activate
(env-6.2.1)drad@thez:/media/pny64/apps/node$ node -v
bash: /media/pny64/apps/node/env-6.2.1/bin/node: Permission denied
(env-6.2.1)drad@thez:/media/pny64/apps/node$
```

**Comment by ekalinin** [comment]
Hey @dradux 

Just tested on the Ubuntu 16.04:

```bash
➥ cd ~/tmp/
➥ virtualenv test-168
➥ . test-168/bin/activate
(test-168) ➥ pip install nodeenv
...
Successfully installed nodeenv-0.13.6

# install in the same virtualenv (build from sources)
(test-168) ➥ nodeenv -p
 * Install node (6.2.1)... done.
 * Appending data to ~/tmp/test-168/bin/activate
(test-168) ➥ node -v
v6.2.1

# install in a new virtualenv (build from sources)
(test-168) ➥ nodeenv -v test-168-X
 * Install node (6.2.1).creating ....
done
(test-168) ➥ deactivate 
➥ . test-168-X/bin/activate 
(test-168-X) ➥ node -v
v6.2.1
```

**Comment by ekalinin** [comment]
Seems, something wrong in your location where you're trying to install.

**Comment by dradux** [comment]
I have found the issue after attempting to create a virtualenv with the nodeenv. The issue is that my partition that I was installing the nodeenv into was mounted with `rw,user,auto` options and the "user" option infers "noexec" which prevents virtualenv or nodeenv from execution. 

The solution is to add the "exec" option and remount the partition. I've tested and verified that a `--prebuilt` and from source both install and a `node -v` returns the nodeenv version after activation.

attribution: http://stackoverflow.com/questions/14759549/strange-permissions-problems-for-virtualenv-python-executable-lubuntu-12-10 (the accepted answer by duncanm)
